ui/app/templates/components/mfa-login-enforcement-header.hbs

```diff
@@ -1,5 +1,5 @@
 {{#if @isInline}}
-  <h3 class="title is-5" data-test-mleh-title>Enforcement</h3>
+  <h3 class="title is-5" data-test-mfa-title>Enforcement</h3>
```

Comment:
I went with the abbreviated component name for acceptance tests when both the header and form components are rendered in the same view to avoid possible collisions. I can tweak if necessary when I get to writing those.

Reply:
ohhh i thought it was a version of "bleh" 😂 I can change these back, I agree that something specific like this is good for avoiding collisions

Reply:
Haha I totally see that now! We created a new slang. _mleh_ is a mix of _meh_ and _blah_ and represents mundane indifference :laughing: